### DISCUSSION TITLE
The Docker type deployment failed, the installation page cannot be opened, please help.

### DISCUSSION THREAD

**Discussion by zidanereal5** [langgenius/dify#4002] [General]
I only modified the mapping port of nginx in the yml file, but cannot open the page when accessing through IP address: 6000/install.
![image](https://github.com/langgenius/dify/assets/123045775/a030a5f3-b8e6-4fb6-9c4d-b87b56f753bb)
![image](https://github.com/langgenius/dify/assets/123045775/6abca881-c2cf-4a58-b76b-e813410c3fdc)

**Comment by zidanereal5** [comment]
![image](https://github.com/langgenius/dify/assets/123045775/0279180a-f28c-4f96-82e3-c664a7a56507)

**Reply by zidanereal5** [reply]
Other configuration is set by default.

**Comment by TinsFox** [comment]
Can you share the complete docker-compose.yml file?

**Comment by patryk20120** [comment]
Shouldn't you try http://localhost:6000/ ?

**Comment by zidanereal5** [comment]
my brower‘s problem, thanks all.
